RELEASE.md

```diff
@@ -55,8 +55,11 @@
     fi;
     ```
   - Make GitHub tags with the release version for each of modules
-    Note- kubera-core-server github tag should be created at last
-    
+    Notes: 
+    - kubera-core-server github tag should be created at last
+    - If build is made from branch, circleci scripts would pick the image name as <branch_name>-ci,
+    - If build is made from git tag, circleci scripts would pick the image name as
```

Comment:
it seems the last words are missing.. it should be 'as git tag'..

Reply:
DOne @vishnuitta